test/source/tests/compose.ts

```diff
@@ -66,6 +66,23 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'compose - strip emojis in a recipient email address',
+      testWithBrowser(async (t, browser) => {
+        const acct = 'flowcrypt.compatibility@gmail.com';
+        await BrowserRecipe.setupCommonAcctWithAttester(t, browser, 'compatibility', {
+          google: { acctAliases: flowcryptCompatibilityAliasList },
+        });
+        const recipientEmail = 'User ⭐ Name <test@email.com>';
+        const msgPwd = 'super hard password for the message';
+        const subject = 'Strip emojis in display name format';
+        const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
+        await ComposePageRecipe.selectFromOption(composePage, acct);
+        await ComposePageRecipe.fillMsg(composePage, { to: recipientEmail }, subject);
+        await ComposePageRecipe.sendAndClose(composePage, { password: msgPwd });
```

Comment:
This test doesn't check if emojis are stripped (as it doesn't fail when we use `Intl.Segmenter`, which doesn't remove emojis), because initial error was on FES side and here we use mock endpoints which doesn't return error when recipient name contains emoji.

I think we should check that name of recipient in sent message doesn't contain emoji (by checking it in `GoogleData` - [example](https://github.com/FlowCrypt/flowcrypt-browser/blob/ab31c96d09b3a2f86f82cfbbbd55fbd5c28cc172/test/source/tests/compose.ts#L1883))

Reply:
Right - that approach will be much more sensible unlike the one I wrote which only confirmed that the issue is fixed when no error occurs during email sent. I will update the test.